src/components/TableCell/TableCell.test.tsx

```diff
@@ -1,42 +1,44 @@
-import { shallow } from "enzyme";
+/* eslint-disable testing-library/no-node-access */
+import { render, isInaccessible } from "@testing-library/react";
 import React from "react";
 
 import TableCell from "./TableCell";
 
 describe("TableCell", () => {
   it("renders", () => {
-    const wrapper = shallow(<TableCell>Test content</TableCell>);
-    expect(wrapper).toMatchSnapshot();
+    const { container } = render(<TableCell>Test content</TableCell>);
+    expect(container.firstChild).toMatchSnapshot();
   });
 
   it("can set a role", () => {
-    const wrapper = shallow(<TableCell role="rowheader" />);
-    expect(wrapper.prop("role")).toEqual("rowheader");
+    const { container } = render(<TableCell role="rowheader" />);
+    expect(container.firstChild).toHaveAttribute("role", "rowheader");
   });
 
   it("can be hidden", () => {
-    const wrapper = shallow(<TableCell hidden={true} />);
-    expect(wrapper.prop("aria-hidden")).toBe(true);
+    const { container, rerender } = render(<TableCell hidden={false} />);
+    expect(isInaccessible(container.firstElementChild)).toEqual(false);
```

Comment:
Using `isInaccessible` to test aria-hidden https://testing-library.com/docs/dom-testing-library/api-accessibility